### DISCUSSION TITLE
Copilot not working in C# VS console app

### DISCUSSION THREAD

**Discussion by marknoble** [community/community#29988] [Copilot Conversations]
I have a few days left on my co-pilot trial and wanted to try it for a quick project.  I installed the Extension and was able to authenticate, but when I tried to use it in my Console app, it didn't seem to do anything. Moreover, several key Copilot options are disabled in the menu; trigger inline suggestion, and open copilot.

![image](https://user-images.githubusercontent.com/3819700/185498916-ef02cfc9-da3d-4124-8c10-75c668bb3e06.png)

I was using it before on another machine and setting it up seemed pretty straightforward. Any idea why it wouldn't be working in this case and why the options would be grayed out?

**Comment by jamesgurung** [comment]
Same issue here on 1.40.0.2
